parquet/pqarrow/encode_arrow_test.go

```diff
@@ -1352,6 +1352,7 @@ var fullTypeList = []arrow.DataType{
 }
 
 var dictEncodingSupportedTypeList = []arrow.DataType{
+	arrow.PrimitiveTypes.Int8,
```

Comment:
I'm not that familiar with the Go package yet, should this list include more of the possible types?

Reply:
Thanks. I like having more tests when they're cheap.